### ISSUE TITLE
parameterType Enum creates Class/TypeAlias 'MyEnum' not found.

### ISSUE THREAD

**Issue by Sicaine** [mybatis/mybatipse#74]
Hi,

i have one sql which takes an Enum as parameterType.

The sql works fine but MyBatis tells me that it can't find the Class/TypeAlias.

Tx.

**Comment by harawata** [comment]
Hi,

Thanks for the report!
But...I couldn't reproduce it.
Both alias and fully-qualified name are correctly recognized.
Here is the enum and the select statement that I used to test.

```java
package test;
public enum MyEnum {
}
```
```xml
<select id="select" parameterType="test.MyEnum">
  select 1
</select>
```
Any difference with your code?

**Comment by Sicaine** [comment]
Tx for your quick replay! :)

I'm using the package element:

`    <typeAliases>
        <package name="test"/>
    </typeAliases>
`

and only using the Name of the Enum itself:
`
    <select id="select" parameterType="MyEnum">
        select 1
    </select>
`

Hope this is reproducable.

**Comment by harawata** [comment]
Still unable to reproduce it. :(

Does using fully-qualified name solve the issue?
If the answer is yes, the plugin couldn't find the configuration file for some reason, I guess.
Check if there is a valid DOCTYPE (no 'www' in the URL).

```xml
<!DOCTYPE configuration
    PUBLIC "-//mybatis.org//DTD Config 3.0//EN"
    "http://mybatis.org/dtd/mybatis-3-config.dtd">
```

In case you need a quick workaround, try [defining custom type alias](https://github.com/mybatis/mybatipse/#type-alias) in the setting.

**Comment by Sicaine** [comment]
Yes the fully-qualified name works. I have, in the same file, another class from the same package and it works fine. It is only the Enum which makes issues.

I will try to reproduce it in a small env, but probably next week.

My mybatis has the typeAliases above the mapper file where my select statements are
Perhaps this file helps you:

```xml
<?xml version="1.0" encoding="UTF-8" ?>
<!DOCTYPE configuration PUBLIC "-//mybatis.org//DTD Config 3.0//EN" "http://mybatis.org/dtd/mybatis-3-config.dtd">
<configuration>
	<settings>
	  ...
	</settings>
    
    <typeAliases>
        ...
		<package name="test"/>
    </typeAliases>
    
    <environments default="standard">
        ...
    </environments>
    
    <mappers>
    	<mapper resource="mapperFile.xml"/>
    </mappers>
</configuration>
```

**Comment by harawata** [comment]
OK, I think I found the cause. Thanks for your help!

**Comment by harawata** [comment]
Hi @Sicaine ,
It should be fixed in 1.1.0. Let me know if there is anything wrong.
Thanks again for your report!

**Comment by Sicaine** [comment]
@harawata works perfect! Got the new version directly through eclipse. Tx :-)
